src/app.tsx

```diff
@@ -70,10 +72,20 @@ export const Application = () => {
     const [alerts, setAlerts] = useState<Alert[]>([]);
     const [rootInfo, setRootInfo] = useState<FileInfo | null>(null);
 
-    const currentPath = decodeURIComponent(options.path?.toString() || "");
+    let currentPath = decodeURIComponent(options.path?.toString() || "/");
+    // Allow users to mess up the path via `?path=`
+    // and always make sure it is a full path like "/home/admin/"
+    if (!currentPath.startsWith('/')) {
+        currentPath = `/${currentPath}`;
+    }
+    if (!currentPath.endsWith('/')) {
+        currentPath = `${currentPath}/`;
```

Comment:
If the given `?path=` argument is not a directory, this gives something invalid. I suppose ENOENT is handled somehow, but what if it's a file, link, etc.?